Source/ManagedNativeWifi/NativeWifi.cs

```diff
@@ -1090,6 +1090,16 @@ internal static bool TryDetectBandChannel(uint frequency, out float band, out in
 				if (factor - Math.Floor(factor) == 0)
 					channel = (int)factor + 34;
 			}
+			else if (frequency is (>= 5_945_000 and <= 7_125_000))
+			{
+				// 6GHz
+				band = 6.0F;
+
+				var gap = frequency / 1_000M - 5_945M; // MHz
+				var factor = gap / 5M;
+				if (factor - Math.Floor(factor) == 0)

```

Comment:
Should there be an else condition that specially considers the possibility of a fractional answer? I always get nervous with a division operation like in line 1099.

1. I've seen floating point variables provide fractional results even if the dividend and divisor should yield no fractional values. Granted, the discrepancy should be extremely minor. Nevertheless, it could throw off the conditional in line 1100 even though it should be interpreted as being equal to 0 for that minuscule difference.
2.  What happens if Windows provides a center frequency that isn't "exactly" like what the tables provide? For example, 5955.2 MHz instead of 5955 MHz. I've seen unexpected values for data rates where they are off just a bit. I don't know if we have to be concerned about that for channel frequency.
3. Should channel width be considered in this calculation? See https://www.electronics-notes.com/articles/connectivity/wifi-ieee-802-11/channels-frequencies-bands-bandwidth.php